.github/ISSUE_TEMPLATE/iceberg_improvement.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: Iceberg Improvement / Feature Request
+description: New features with Apache Iceberg
+labels: ["kind:new feature"]
+body:
+  - type: textarea
+    attributes:
+      label: Feature Request / Improvement
+      description: Please describe the feature and elaborate on the use case and motivation behind it
+    validations:
+      required: true
```

Comment:
We may want an optional query engine input here as well, since it could be for a specific engine.

Reply:
I like it, thanks!